src/modules/auth/auth.controller.ts

```diff
@@ -1,26 +1,25 @@
-import {Body, Controller, HttpStatus, Post, UseGuards} from '@nestjs/common';
-import {ApiResponse, ApiTags} from '@nestjs/swagger';
-import {AuthGuard} from '@nestjs/passport';
+import { Body, Controller, HttpStatus, Post, UseGuards } from '@nestjs/common';
```

Comment:
sorry for the formatting changes. I deactivated my prettier.

Reply:
Should we create some Linting rules and go with that?